5.sc_Morphology_Logistic_Regression/0.split_data/split_data.ipynb

```diff
@@ -6,60 +6,55 @@
    "id": "2a0c7e4f",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # save indexes as tsv file
</pre><p>Do you save the indices because the dataset is large? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Yes, easier to manage reloading data between analysis files too.</p>